contracts/contracts/core/vault/src/main.sw

```diff
@@ -14,6 +14,7 @@ __     __          _ _
 */
 
 use std::{
+    asset::{mint_to, burn},
```

Comment:
We should be using ownership from sway-libs instead of custom gov implementation